use-timescale/data-tiering/about-data-tiering.md

```diff
@@ -9,18 +9,18 @@ cloud_ui:
         - [services, :serviceId, overview]
 ---
 
-import TieredStorageBilling from "versionContent/_partials/_tiered-storage-billing.mdx";
-
 # About the object storage tier
 
-Timescale's tiered storage architecture includes a standard high-performance storage tier and a low-cost object storage tier built on Amazon S3. You can use the standard tier for data that requires quick access, and the object tier for rarely used historical data. Chunks from a single hypertable, including compressed chunks, can stretch across these two storage tiers. A compressed chunk uses a different storage representation after tiering.
+$COMPANY's tiered storage architecture includes a standard high-performance storage tier and a low-cost object storage tier built on Amazon S3. You can use the standard tier for data that requires quick access, and the object tier for rarely used historical data. Chunks from a single hypertable, including compressed chunks, can stretch across these two storage tiers. 
 
-In the high-performance storage, chunks are stored in the block format. In the object storage, they are stored in a compressed, columnar S3 format. This format is different from that of the internals of your $SERVICE_LONG. It allows for more efficient columnar scans across longer time periods, and $CLOUD uses other metadata and query optimizations to reduce the amount of data that needs to be fetched from the object storage tier to satisfy a query.
+In the high-performance storage, your data is stored in the block format. In the object storage, it is stored in [Apache Parquet][parquet]. The original size of the data in your $SERVICE_SHORT, compressed or uncompressed, does not correspond directly to its size in S3. A compressed hypertable may even take more space in S3 than it does in $CLOUD_LONG.
 
-<TieredStorageBilling />
+The columnar storage format allows for more efficient scans across longer time periods, and $CLOUD_LONG uses other metadata and query optimizations to reduce the amount of data that needs to be fetched from the object storage tier to satisfy a query.
```

Comment:
> The columnar storage format

You mean Parquet as format? This is the first instance of "columnar storage format", so I'd suggest to reword it to:

```suggestion
Parquet is a columnar storage format that allows for more efficient scans across longer time periods, and $CLOUD_LONG uses other metadata and query optimizations to reduce the amount of data that needs to be fetched from the object storage tier to satisfy a query.
```

Reply:
true, updated!

Reply:
Thank you, I already approved the PR, no objection from my side to merge.